include/paseto.h

```diff
@@ -17,6 +17,26 @@ extern "C"{
 #define paseto_v2_PUBLIC_PUBLICKEYBYTES 32U
 #define paseto_v2_PUBLIC_SECRETKEYBYTES 64U
 
+enum paseto_key_header {
+    V2_LOCAL = 4,
+    V2_PUBLIC = 5
```

Comment:
Everything in the header should be namespaced/prefixed so it doesn't potentially collide with other libraries or code. Enums aren't namespaced in C(++).

Reply:
Understood. We don't normally do C development and was hoping the CI pipeline would catch these bugs.